README.rst

```diff
@@ -141,6 +141,10 @@ garbage collection.
 **B021**: f-string used as docstring. This will be interpreted by python
 as a joined string rather than a docstring.
 
+**B022**: No arguments passed to `contextlib.suppress`.
+No exceptions will be suppressed and therefore this context manager is redundant.
```

Comment:
Maybe we could note here our miss of `from contextlib import suppress` and the reason why - noise as we're sure other people call methods suppress ...

Reply:
Good shout, will add a note 👍

Reply:
I've added that in now